src/utils/telemetry/report-error.mjs

```diff
@@ -30,15 +31,17 @@ export const reportError = async function (error, config = {}) {
       user: {
         id: globalConfig.get('userId'),
       },
+      metadata: config.metadata,
       osName: process.platform,
       cliVersion,
       nodejsVersion,
     },
   })
 
-  // spawn detached child process to handle send
-  execa(process.execPath, [join(dirPath, 'request.mjs'), options], {
+  // spawn detached child process to handle send and wait for the http request to finish
+  // otherwise it can get canceled
+  await execa(process.execPath, [join(dirPath, 'request.mjs'), options], {
```

Comment:
Will this block until the http request finishes?

Reply:
yea it needs to because if it's called in the error reporting the process is otherwise exiting immediately which leads to the function crashing as the TLS handshake could not be finished:

https://app.netlify.com/sites/cli/functions/error-reporting
![CleanShot 2023-04-20 at 09 27 56](https://user-images.githubusercontent.com/11156362/233292289-a1a68954-008a-4c3e-a7f1-8589563b85d7.png)

Reply:
Interesting, was this happening before removing the `unref` method call?

Reply:
yep exactly that's the reason why I've changed it